source/includes/items/_create.md

```diff
@@ -0,0 +1,342 @@
+        "population": [],
+        "address": "sdfefs, esf, esf, esf, ffff",
+        "floorplan": {
+          "id": 6,
+          "location_id": 22,
+          "folder_id": 22,
+          "file": {
+            "url": "https://awesome-assets-bucket.s3.amazonaws.com/development/uploads/floorplan/file/6/20210217_081919.jpg",
+            "thumb": {
+              "url": "https://awesome-assets-bucket.s3.amazonaws.com/development/uploads/floorplan/file/6/thumb_20210217_081919_thumb.png"
+            },
+            "png": {
+              "url": "https://awesome-assets-bucket.s3.amazonaws.com/development/uploads/floorplan/file/6/png_20210217_081919_png.png"
+            }
+          },
+          "width": 3264,
+          "height": 2448,
+          "main": true,
+          "uuid": "a8e2eacb-9d4b-456a-a101-f19fc0fa53fd",
+          "created_at": "2021-04-09T16:42:51.000+01:00",
+          "updated_at": "2021-04-09T16:42:51.000+01:00",
+          "file_tmp": null
+        }
+      }
+    },
+    "condition": {
+      "id": 3,
+      "name": "poor",
+      "label": "Poor",
+      "description": null,
+      "uuid": "cb274607-0af0-40a1-96c9-3324d0639982",
+      "created_at": "2020-11-09T13:08:36.000+00:00",
+      "updated_at": "2020-11-09T13:08:36.000+00:00",
+      "client_id": null,
+      "colour": null,
+      "is_default": false
+    },
+    "status": {
+      "id": 23,
+      "name": "waiting_regeneration",
+      "label": "Awaiting Regeneration",
+      "description": null,
+      "status_type": null,
+      "uuid": "69bd2c6f-7311-421c-b956-4c55955ebe63",
+      "created_at": "2020-11-09T13:08:36.000+00:00",
+      "updated_at": "2020-11-09T13:08:36.000+00:00",
+      "reportable": true,
+      "reprocessable": false,
+      "archivable": false,
+      "icon_name": null,
+      "icon_family": null,
+      "icon": {
+        "url": null
+      },
+      "width": null,
+      "height": null,
+      "client_id": null,
+      "is_default": false,
+      "reusable": false
+    },
+    "notes": [],
+    "task_items": [],
+    "item_attributes": [],
+    "stamps": []
+  },
+  "status": "created"
+}
+```
+
+This endpoint creates a new Item.
+
+### HTTP Request
+
+`POST http://localhost:3000/api/v6/items`
+
+### Item Parameters
+
+Parameter | Default | Description | Type | Required? | Options
+--------- | ------- | ----------- | ---- | --------- | -------
+location_id | nil | The ID of the location of the item | Integer | Yes
+condition_id | nil | The ID of the item's condition | Integer | Yes | See the Conditions endpoint
+status_id | nil | The ID of the item's status | Integer | Yes | See the Statuses endpoint
+code | nil | A free text code identifier | String | Yes
+usage | nil | Whether the Item is in use or unused | String | Yes | 'in_use', 'unused'
+project | nil | A free text project identifier | String | No
+x | nil | ??? The x position of the item on its location's floorplan. pixels or 0 to 1? ??? | Decimal | No
```

Comment:
are coordinates relevant to ISS? and are coordinates related to pixels, or are they proportional (o to 1, or -1 to 1 or something)

Reply:
Coordinates are likely not needed by ISS as they wont be using the same floorplan (or at least floorplan ratio) as us. But it should still remain in the docs, i believe the value it's position as a percentage of the floorplan (so 0.0 to 1.0).